packages/eslint-plugin/docs/rules/no-unnecessary-type-constraint.md

```diff
@@ -19,8 +19,12 @@ It is therefore redundant to `extend` from `any`.
 ```ts
 interface FooAny<T extends any> {}
 
+interface FooUnknown<T extends unknown> {}
+
 type BarAny<T extends any> = {};
 
+type BarUnknown<T extends unknown> = {};
```

Comment:
[Praise] Just confirming, yes, this looks reasonable to me 😄. We want both the existing `extends any`s and `extends unknown`s.

Reply:
:smiley: